cfgov/cfgov/settings/base.py

```diff
@@ -34,14 +35,30 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.humanize',
     'storages',
-    'localflavor',
     'flags',
     'v1',
     'core',
     'sheerlike',
 )
 
+OPTIONAL_APPS=[
+    {'import':'noticeandcomment','apps':('noticeandcomment','cfpb_common')},
+    {'import':'cfpb_common','apps':('cfpb_common','cfpb_common')},
+    {'import':'jobmanager','apps':('jobmanager','cfpb_common')},
+    {'import':'cal','apps':('cal','cfpb_common')},
+    {'import':'comparisontool','apps':('comparisontool','haystack','cfpb_common')},
+    {'import':'agreements','apps':('agreements','haystack', 'cfpb_common')},
+    {'import':'knowledgebase','apps':('knowledgebase','haystack', 'cfpb_common')},
```

Comment:
does this syntax mean, include `knowledgebase` that depends on `haystack` & `cfpb_common`